Services/ProjectService/Startup.cs

```diff
@@ -38,37 +41,33 @@ public void ConfigureServices(IServiceCollection services)
             });
 
             services.AddControllers();
-
+            services.AddKernelExtensions(Configuration);
             ConfigureCommands(services);
             ConfigureRepositories(services);
             ConfigureMappers(services);
             ConfigureValidators(services);
             ConfigureMassTransit(services);
-            services.AddMassTransitHostedService();
         }
 
         private void ConfigureMassTransit(IServiceCollection services)
         {
-            services.AddMassTransit(configurator =>
+            services.AddMassTransit(x =>
             {
-                configurator.UsingRabbitMq((context, factoryConfigurator) =>
+                x.UsingRabbitMq((context, cfg) =>
                 {
-                    const string serviceInfoSection = "ServiceInfo";
-
-                    var serviceName = Configuration.GetSection(serviceInfoSection)["Name"];
-                    var serviceId = Configuration.GetSection(serviceInfoSection)["ID"];
-
-                    factoryConfigurator.Host("localhost", hostConfigurator =>
+                    cfg.Host(options.Host, host =>
                     {
-                        hostConfigurator.Username($"{serviceName}_{serviceId}");
-                        hostConfigurator.Password(serviceId);
+                        host.Username(options.Username);
+                        host.Password(options.Password);
                     });
                 });
 
-                configurator.AddRequestClient<IGetFileRequest>(
-                    new Uri("rabbitmq://localhost/FileService"));
+				x.AddRequestClient<IGetFileRequest>(
+					new Uri("rabbitmq://localhost/FileService"));
             });
-        }
+
+            services.AddMassTransitHostedService();
+		}
 
         private void ConfigureCommands(IServiceCollection services)
```

Comment:
Fix spaces.

Reply:
What should I fix? Is this really important? jesus
![image](https://user-images.githubusercontent.com/61093487/91717988-3af7d480-eb9b-11ea-91f2-6ad9587759fa.png)